### PR TITLE
move /openid/server to DW::Controller

### PR DESCRIPTION
**Pull request by kareila**
I removed the code block concerning fake `_ljwho` identities and OPENID_DEST_DOMAIN_TRUSTED hashes, because it seemed outdated and unused.

This is a new multiple-use branch to gradually convert pages under htdocs/openid.  I did this one first because it was simplest, but without a functioning dreamhack, I don't have a way to test it right now.

### CODE REVIEW

File: cgi-bin/DW/Controller/OpenID.pm [hunk 1]
```diff
@@ -22,10 +23,41 @@ use DW::Routing;
...
+    my $r = DW::Request->get;
+    my $get = $r->get_args;
+
+    my $trust_root = $get->{'openid.trust_root'} // '';
+    my $return_to  = $get->{'openid.return_to'} // '';
+
+    ## Non-OpenID-compliant section: rewrite LiveJournal's trust_root to
+    ## https so that it will match their return_to URL and pass validation.
+    $get->{'openid.trust_root'} = 'https://www.livejournal.com/'
+        if ( $trust_root eq 'http://www.livejournal.com/' &&
```
Comment (zorkian) [inline cgi-bin/DW/Controller/OpenID.pm]: I believe the style guide requests not to use parenthesis when they aren't necessary?

**Comment by zorkian** [comment]
Yeah I forget what that was for but it might have been Fotobilder?

File: cgi-bin/DW/Controller/OpenID.pm (hunk 1 above)
Comment (kareila) [inline cgi-bin/DW/Controller/OpenID.pm]: I plead copy/paste artifact.

**Comment by kareila** [comment]
Whatever it was, it's gone now.

**Pull request merged**